<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>rm_dir</class>
 <widget class="QDialog" name="rm_dir">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>571</width>
    <height>446</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>删除指定文件</string>
  </property>
  <property name="styleSheet">
   <string notr="true">*{
	font-size:18px;
}
QPushButton{
	color:rgb(255, 255, 255);
	background-color:rgb(0, 170, 255)
}</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout" stretch="1,1,1,1,0,1">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>请选择要递归描的目录:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_dir">
       <property name="text">
        <string>请选择目录</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>已选择目录:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="lineEdit_dir"/>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QPushButton" name="pushButton_scan">
     <property name="text">
      <string>递归扫描文件夹</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="label_3">
     <property name="text">
      <string>扫描结果:</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QListWidget" name="listWidget"/>
   </item>
   <item>
    <widget class="QPushButton" name="pushButton_remove">
     <property name="text">
      <string>删除选中的文件</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
